pkg/account/service.go

```diff
@@ -124,7 +124,7 @@ func (a *Service) Update(ID string, data *Account) (*Account, error) {
 		return nil, err
 	}
 
-	err = mergo.Merge(account, *data)
+	err = mergo.Merge(account, *data, mergo.WithOverride)
```

Comment:
This was the whole bug right here. `mergo.Merge()` does not override values in the src struct by default.